packages/react-form/src/hooks/field/test/field.test.tsx

```diff
@@ -292,6 +294,86 @@ describe('useField', () => {
     });
   });
 
+  describe('#reduceField', () => {
+    function buildState<Value>({
+      value,
+      defaultValue,
+      dirty = false,
+    }): FieldState<Value> {
+      return {
+        value,
+        defaultValue,
+        error: undefined,
+        touched: true,
+        dirty,
+      };
+    }
+
+    describe('when the new value is the same as the default value', () => {
```

Comment:
These tests are OK, and they do seem to be testing the right behaviour, but I'd prefer if they were testing the public API (`useField`) instead of `reduceField` specifically, were there any technical reasons that we couldn't do that?

Reply:
I started down that road at first and switched to these tests because putting the tests in `useField` required a lot of setup; more than I thought was reasonable.